packages/types/lib/integration/template.ts

```diff
@@ -68,6 +68,7 @@ export interface TemplateOAuth2 extends Template {
 
     refresh_url?: string;
     expires_in_unit?: 'milliseconds';
+    expires_in_mins?: number; //in minutes
```

Comment:
expires in ms would provide more flexibility and is a more standard unit imo

Reply:
I have updated this in the latest [commit](https://github.com/NangoHQ/nango/pull/2601/commits/b0bbeae89689814d76feb9461bd89d21b777912a).